test/test.js

```diff
@@ -44,6 +44,21 @@ const responses = {
   }
 }
 
+const cleanDOM = () => {
```

Comment:
This can be Mocha's per-suite `teardown` block instead of defining it as a function and invoking it at the end of every test.